sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTest.java

```diff
@@ -437,6 +445,28 @@ public void listCertificates(HttpClient httpClient, CertificateServiceVersion se
         });
     }
 
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getTestParameters")
+    public void listPropertyOfCertificatesThroughPoller(HttpClient httpClient, CertificateServiceVersion serviceVersion) {
+        createCertificateClient(httpClient, serviceVersion);
+        listPropertyOfCertificatesRunner((certificates) -> {
```

Comment:
I suggested the `throughPoller` suffix for the first `cancelCertificateOperation()`, since it actually uses the actual Poller object to cancel the operation. This suffix is not necessary here because we are simply calling `listPropertiesOfCertificates()`. I would still rename it to that exact name instead of having it deviate to `listPropertyOfCertificates()`.
```suggestion
    public void listPropertiesOfCertificates(HttpClient httpClient, CertificateServiceVersion serviceVersion) {
        createCertificateClient(httpClient, serviceVersion);
        listPropertiesOfCertificatesRunner((certificates) -> {
```